src/controllers/album.js

```diff
@@ -0,0 +1,58 @@
+const db = require('../services/db');
```

Comment:
Youre importing the same thing twice here under a different name, then not using one of them. Should delete this first one